editor/action_map_editor.cpp

```diff
@@ -56,6 +56,14 @@ void ActionMapEditor::_event_config_confirmed() {
 	Dictionary new_action = current_action.duplicate();
 	Array events = new_action["events"].duplicate();
```

Comment:
Question for the editor team - why do we duplicate events here? If we didn't, it could potentially optimize it when many events are on one action, though unlikely.

Reply:
Updating this in place would break undo/redo